src/Loader.php

```diff
@@ -231,7 +232,7 @@ protected function sanitiseVariableValue($name, $value)
 
             // Unquoted values cannot contain whitespace
             if (preg_match('/\s+/', $value) > 0) {
-                throw new InvalidArgumentException('Dotenv values containing spaces must be surrounded by quotes.');
+                throw new SyntaxException();
```

Comment:
There will be other type syntax exception future. 
```
'Dotenv values containing spaces must be surrounded by quotes.'
```
is not a base exception message. Using it as default is weird. 
How about making error message required parameter of SyntaxException construstor ?